TLM/TLM/Manager/Impl/AdvancedParkingManager.cs

```diff
@@ -2460,7 +2468,12 @@ bool citizenDebug
                 return true;
             }
 
-            LoopUtil.SpiralLoop(centerI, centerJ, radius, radius, LoopHandler);
+            for (int i = 0; i < _spiralGridCoordsCache.Length; i++) {
```

Comment:
Though about that during the implementation but decided against it since this is perf critical code and I don't see a way to do what you say without additional mem allocations. I could be wrong though, maybe you have a suggestion of how the implementation could look like?

Reply:
I'm not skilled enough yet to avoid mistakes so don't have any suggestions for implementation. Just don't like duplicate code. In any case, this is not a blocking issue.